jib-core/src/main/java/com/google/cloud/tools/jib/cache/LayerEntriesSelector.java

```diff
@@ -82,24 +96,26 @@ public boolean equals(Object other) {
       }
       LayerEntryTemplate otherLayerEntryTemplate = (LayerEntryTemplate) other;
       return sourceFile.equals(otherLayerEntryTemplate.sourceFile)
-          && extractionPath.equals(otherLayerEntryTemplate.extractionPath);
+          && extractionPath.equals(otherLayerEntryTemplate.extractionPath)
+          && lastModifiedTime.equals(otherLayerEntryTemplate.lastModifiedTime);
     }
 
     @Override
     public int hashCode() {
-      return Objects.hash(sourceFile, extractionPath);
+      return Objects.hash(sourceFile, extractionPath, extractionPath);
```

Comment:
Why is `extractionPath`  here twice?  Should the second be `lastModifiedTime`?

Reply:
oops